tests/cfg/TEMPORARY_zalrsc.yaml

```diff
@@ -1,10 +1,11 @@
 name: TEMPORARY_zalrsc
 description: Enables the ZALRSC extension, and disables the zcmp extentnion. TODO, krdosvik, remove this configuration when no problem running zalrsc test with ISS=1, so until then run with ISS=0.
 compile_flags:
-    +define+ZALRSC
+    +define+A
+    #+define+ZALRSC TODO: use zalrsc when ISS is up to date.
```

Comment:
My interpretation:
We only need Zalrsc, but there be ISS troubles, so we enable A which is technically overkill but gives us what we want.

Seems reasonable.